3_javascript/javascript1/week2/homework2ndweek/fullname.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Flight booking</title>
+  </head>
+  <body>
+    <script>
+      const firstname1 = getFullName("Monika", "Marciniak");
```

Comment:
`firstName` and `lastName` are variables that should be passed to the function `getFullName` as paarmeters.

Reply:
const firstname1 = getFullName(firstName, surName); Like that?

Reply:
I mean the const name should be `fullName1` and `fullName2`